packages/web-components/fast-foundation/src/select/select.template.ts

```diff
@@ -46,17 +50,17 @@ export function selectTemplate(
                         ${ref("control")}
                     >
                         ${startSlotTemplate(options)}
-                        <slot name="button-container">
+                        <slot name="control">
                             <div class="selected-value" part="selected-value">
                                 <slot name="selected-value">${x => x.displayValue}</slot>
                             </div>
-                            <div aria-hidden="true" class="indicator" part="indicator">
-                                <slot name="indicator">
-                                    ${options.indicator || ""}
-                                </slot>
-                            </div>
                         </slot>
                         ${endSlotTemplate(options)}
+                        <div aria-hidden="true" class="open-icon" part="open-icon">
+                            <slot name="open-icon">
+                                ${options.openIcon || openIcon}
+                            </slot>
+                        </div>
```

Comment:
This is complex because the open icon should technically be replaced if one replaces the invoking control. We might need to spend a bit more time considering this.

Reply:
Same comment here on `open` as well.

Reply:
See above: let's not hard-link the default icon.

Reply:
The `button-container` is added back, which replaces the invoking control including this icon.